cli/app.go

```diff
@@ -970,14 +970,6 @@ After creation, use 'viam module update' to push your new module to app.viam.com
 							Value:     "./meta.json",
 							TakesFile: true,
 						},
-						&cli.StringFlag{
-							Name:  moduleFlagPublicNamespace,
-							Usage: "the public namespace where the module resides (alternative way of specifying the org id)",
-						},
-						&cli.StringFlag{
-							Name:  moduleFlagOrgID,
-							Usage: "id of the organization that hosts the module",
-						},
```

Comment:
These flags don't sense for update because the meta.json already has the module_id. If anyone was passing these in, I suspect it was a bug. 

Thoughts?

Reply:
Agreed. They are needed in upload, but don't make sense for update. I'll send an email with the scope change.